phpunit/fonts-library/class-wp-fonts-library-test.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Tests for Fonts Library class
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ */
+
+/**
+ * @coversDefaultClass WP_Fonts_Library
+ */
+class WP_Fonts_Library_Test extends WP_UnitTestCase {
+
+	/**
+	 * @covers ::get_fonts_dir
+	 */
+	public function test_get_fonts_dir() {
+		$this->assertStringEndsWith( '/wp-content/uploads/fonts', WP_Fonts_Library::get_fonts_dir() );
+	}
+
+	/**
+	 * @covers ::set_upload_dir
+	 *
+	 * @dataProvider data_set_upload_dir
+	 *
+	 * @param array $defaults Default upload directory data.
+	 * @param array $xpected Modified upload directory data.
+	 */
+	public function test_set_upload_dir( $defaults, $xpected ) {
+		$this->assertEquals( $xpected, WP_Fonts_Library::set_upload_dir( $defaults ) );
```

Comment:
```suggestion
	 * @param array $defaults Default upload directory data.
	 * @param array $expected Modified upload directory data.
	 */
	public function test_set_upload_dir( $defaults, $expected ) {
		$this->assertSame( $expected, WP_Fonts_Library::set_upload_dir( $defaults ) );
```
- Minor typo correction from `$xpected` to `$expected`.
- Changed `assertEquals()` to `assertSame()`.